### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by nicarq**
Enjoy!

### CODE REVIEW

**Comment by SebastienGllmt** [comment]
Note: This will conflict with #421

**Comment by nicarq** [comment]
@SebastienGllmt yeah. we missed sync about this. I will write a `README` on how to add a new language so it’s always added with the right name

**Comment by SebastienGllmt** [comment]
@nicarq looks like the locale mapping didn't work for the ToS. You can see because this PR includes `zh-CN.md` and `zh-TW.md` instead of `zh-Hans` and `zh-Hant` as expected.

File: app/i18n/locales/terms-of-use/ada/zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# 服务协议条款
```
Comment (SebastienGllmt) [inline app/i18n/locales/terms-of-use/ada/zh-CN.md]: Example of file that didn't get its locale mapped.

Comment (nicarq) [reply app/i18n/locales/terms-of-use/ada/zh-CN.md]: the changes probably need to be done in another PR to sync Crowdin

**Pull request merged**